index.js

```diff
@@ -137,6 +142,52 @@ const showAbout = aboutWindow => () => {
   }
 };
 
+/**
+ * This command `getUserDefault` is like doing this in your OSX Terminal:
+ *   defaults read -g AppleFirstWeekday
+ *
+ * If all is ok should return something like this:
+ *   { gregorian = 2; }
+ *
+ * But can return a not found:
+ *   `The domain/default pair of
+ *    (kCFPreferencesAnyApplication, AppleFirstWeekday) does not exist`
+ *
+ * We ignore it when there is an error:
+ * If setting first day on Apple UI doesn't work. They can do this:
+ * in command line:
+ *   defaults write NSGlobalDomain AppleFirstWeekday -dict 'gregorian' 2
+ *
+ * This will setup first day of week to monday
+ */
+function readOSXUserFirstWeekDay () {
+  const firstWeekdayPref = systemPreferences.getUserDefault(
+    'AppleFirstWeekday',
+    'dictionary'
+  );
+
+  if (!Object.keys(firstWeekdayPref).length) return null
+
+  // key in this example is `gregorian` and day is `2`
+  // { gregorian = 2; }
+  const key = Object.keys(firstWeekdayPref)[0]
+  return firstWeekdayPref[key]
```

Comment:
Can this just be `return firstWeekdayPref.gregorian`, or does this change if the user doesn't use a gregorian calendar?

Reply:
This way we don't care for the key, we just want the value which the number of the first day of week